app/main.py

```diff
@@ -230,6 +206,38 @@ def contact():
 @app.route("/terms")
 def terms():
     return render_template('terms.html', page_class="terms")
+
+
+@app.route("/api/parties")
+def parties_api():
+    db_conn = psycopg2.connect(DATABASE_URL)
+    db_cur = db_conn.cursor(cursor_factory=psycopg2.extras.DictCursor)
+    db_cur.execute('SELECT * from parties')
```

Comment:
spotted below that by doing `SELECT * from ...` we are getting all the database columns including the password hash.

I would sugges that we don't use `*` here but instead use the names of the columns we are interested in explicitly.